smpp/pdu/pdufield/tlvtypes.go

```diff
@@ -0,0 +1,431 @@
+package pdufield
+
+// BindTransmiterTLVParameter defines the TLV parameters available  for BindTrasmiter PDU
+var BindTransmiterTLVParameter map[string]TLVType
+
+// SubmitSMTLVParameter defines the TLV parameters available for SubmitSM PDU
+var SubmitSMTLVParameter map[string]TLVType
+
+// SubmitSMMultiTLVParameter defines the TLV parameters available for submitSMMulti PDU
+var SubmitSMMultiTLVParameter map[string]TLVType
+
+// DeliverSMTLVParameter defines TLV parameters available for DeliverSM PDU
+var DeliverSMTLVParameter map[string]TLVType
+
+//DataSMTLVParameter defines the TLV parameters available for DataSM PDU
+var DataSMTLVParameter map[string]TLVType
+
+// DataSMRespTLVParameter defines the TLV parameters available for DataSMResp PDU
+var DataSMRespTLVParameter map[string]TLVType
+
+func init() {
```

Comment:
Yes, so when dependencies try to set a TLV they use that constant instead of strings.